docs/examples/custom_operations/custom_operator/create_a_custom_operator.ipynb

```diff
@@ -4,30 +4,33 @@
    "cell_type": "markdown",
```

Comment:
<p>Prerequisites is not rendered?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2568/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>